web/models/user.ex

```diff
@@ -6,6 +6,7 @@ defmodule Shareguru.User do
     field :email, :string
     field :picture, :string
     field :expire_at, Ecto.DateTime
+    field :hd, :string, virtual: true # Mail domain
```

Comment:
That's the whole problem - `hd` is coming from Google API and this is domain of mail. There are 2 things how we could handle that:

1. Same as is done right now - hd is virtual attribute so it's not saved anywhere.
2. Translate `hd` key from `params` to smth more meanigful (maybe create special facade that would make this?)

What do you think? When I think right now I prefer option 2 tbh. We won't have bloated model then :)